mixins/localize/test/localize-mixin.test.js

```diff
@@ -308,17 +309,17 @@ describe('LocalizeMixin', () => {
 				expect(val).to.equal('Hello Bill');
 			});
 
-			it('should re-localize text when locale changes', (done) => {
+			it('should re-localize text when locale changes', () => {
 				const valInitial = elem.localize('hello', { name: 'Sam' });
 				expect(valInitial).to.equal('Hello Sam');
-				const myEventListener = () => {
+				let resolve;
+				elem.addEventListener('d2l-localize-resources-change', () => {
 					const val = elem.localize('hello', { name: 'Mary' });
 					expect(val).to.equal('Bonjour Mary');
-					elem.removeEventListener('d2l-localize-resources-change', myEventListener);
-					done();
-				};
-				elem.addEventListener('d2l-localize-resources-change', myEventListener);
+					resolve();
+				}, { once: true });
 				documentLocaleSettings.language = 'fr';
+				return new Promise(r => resolve = r);
```

Comment:
This was all the result of flailing trying to get tests to pass. I can put this back how it was.

Reply:
Cool -- although the `once: true` is a nice add.